### ISSUE TITLE
NTR: Myeloperoxidase activity low in neutrophils and monocytes (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8352]
New term request Myeloperoxidase activity low in neutrophils and monocytes (elthat_130621132516):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000375
- label: Reduced neutrophil myeloperoxidase activity
- parents: [Abnormal leukocyte enzyme concentration or activity (HP:0034858)](https://hpo.jax.org/app/browse/term/HP:0034858)
- pmid: PMID:5796360
- def: Concentration or activity of myeloperoxidase (EC 1.11.1.7) as measured in neutrophils is below the limits of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8352
- synonyms:
